ignite-base/App/Redux/CreateStore.js

```diff
@@ -45,8 +45,9 @@ export default (rootReducer, rootSaga) => {
     enhancers.push(autoRehydrate())
   }
 
-  // in dev mode, we'll create the store through Reactotron
-  const createAppropriateStore = __DEV__ ? console.tron.createStore : createStore
+  // if Reactotron is enabled (default for __DEV__), we'll create the store through Reactotron
+  const USE_REACTOTRON = Config.reactotron
+  const createAppropriateStore = USE_REACTOTRON ? console.tron.createStore : createStore
```

Comment:
why make a const if it's only going to be used this once?  maybe just use `Config.reactotron` here?  I don't think it adds readability